gradle.properties

```diff
@@ -12,11 +12,11 @@
 dcosAddress=master.mesos
 
 ### dependencies
-pravegaVersion=0.2.1
-connectorVersion=0.2.1
+pravegaVersion=0.3.0-SNAPSHOT
```

Comment:
can we use a snapshot from jfrog instead of using a locally build snapshot?

Reply:
Done. I also added the jfrog repository link in the `gradle.build` file.